hack/install.sh

```diff
@@ -529,6 +529,19 @@ do_install() {
 	exit 1
 }
 
+while getopts "e:" opt; do
+	case $opt in
+		e)
+			environment=$OPTARG
```

Comment:
This variable ought to be initialized before this loop starts to ensure we don't get side effects from the enclosing shell, ie `environment=` on a line by itself right before the `while` starts.